src/encoded/static/components/matrix_reference_epigenome.js

```diff
@@ -415,6 +411,92 @@ MatrixHeader.contextTypes = {
     navigate: PropTypes.func,
 };
 
+const ClickableThumbnail = (props) => {
```

Comment:
There is something with the name that I cannot point my finger at. I know what it means but at the same time I cannot describe what it means based solely on the function name.

Reply:
Hm, let me know if you have an idea of something to change it to. It's a thumbnail (small image) which is clickable (expandable) so that's why I called it this.